src/applications/lgy/coe/containers/introduction-content/COEStatuses/COEDenied.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+
+const COEDenied = () => (
+  <>
+    <va-alert status="info">
+      <h3 slot="headline">We denied your request for a COE</h3>
```

Comment:
I think this should be an h2 since the preceding header is an h1

Reply:
Updated